google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITGrpcTest.java

```diff
@@ -258,5 +258,6 @@ public void testGrpcUniverseDomainMatchesHost() throws Exception {
         () ->
             assertThat(
                 storage.getOptions().getHost().equals("https://storage.my-universe-domain.com")));
+    storage.close();
```

Comment:
I think this test can be changed to stop at the options piece. Its assertion the host from the options in the storage instance.

```
  @Test
  public void testGrpcUniverseDomainMatchesHost() throws Exception {
    StorageOptions options =
        StorageOptions.grpc().setUniverseDomain("my-universe-domain.com").build();
    assertAll(
        () -> assertThat(options.getUniverseDomain().equals("my-universe-domain.com")),
        () -> assertThat(options.getHost().equals("https://storage.my-universe-domain.com")));
  }
```

Also, if one of the assertions failed, we wouldn't make it to this close. If we were keeping the storage instance around, we should wrap it in a try-with-resource to auto close it.